### PR TITLE
Complete hls-hlint-plugin package metadata

### PR DESCRIPTION
**Pull request by jneira**
Maybe we should update the `author` field (and others)
//cc @alanz @ndmitchell

### CODE REVIEW

File: plugins/hls-hlint-plugin/hls-hlint-plugin.cabal [hunk 1]
```diff
@@ -2,11 +2,11 @@ cabal-version: 2.2
 name:          hls-hlint-plugin
 version:       0.1.0.0
 synopsis:      Hlint integration plugin with Haskell Language Server
-description:   Please see README.md
+description:   Please see Haskell Language Server Readme (https://github.com/haskell/haskell-language-server/blob/master/README.md)
 license:       Apache-2.0
 license-file:  LICENSE
 author:        Many,TBD when we release
-maintainer:    alan.zimm@gmail.com (for now)
+maintainer:    alan.zimm@gmail.com
```
Comment (pepeiborra) [inline plugins/hls-hlint-plugin/hls-hlint-plugin.cabal:9]: Shouldn't you be listed as the maintainer too @jneira ?

**Review by pepeiborra — APPROVED** [review]

File: plugins/hls-hlint-plugin/hls-hlint-plugin.cabal [hunk 2]
```diff
@@ -2,11 +2,11 @@ cabal-version: 2.2
 name:          hls-hlint-plugin
 version:       0.1.0.0
 synopsis:      Hlint integration plugin with Haskell Language Server
-description:   Please see README.md
+description:   Please see Haskell Language Server Readme (https://github.com/haskell/haskell-language-server/blob/master/README.md)
```
Comment (ndmitchell) [inline plugins/hls-hlint-plugin/hls-hlint-plugin.cabal:5]: https://github.com/haskell/haskell-language-server#readme is what i usually use

**Review by ndmitchell — APPROVED** [review]

File: plugins/hls-hlint-plugin/hls-hlint-plugin.cabal (hunk 1 above)
Comment (alanz) [reply plugins/hls-hlint-plugin/hls-hlint-plugin.cabal:9]: We probably need some sort of generic reference to the hls team. Is that possible?

File: plugins/hls-hlint-plugin/hls-hlint-plugin.cabal (hunk 2 above)
Comment (jneira) [reply plugins/hls-hlint-plugin/hls-hlint-plugin.cabal:5]: oh, yeah, thanks

File: plugins/hls-hlint-plugin/hls-hlint-plugin.cabal (hunk 1 above)
Comment (jneira) [reply plugins/hls-hlint-plugin/hls-hlint-plugin.cabal:9]: I would prefer some collective thing too, but i am not against include myself if needed, of course.
I guess we would need a generic email account.

File: plugins/hls-hlint-plugin/hls-hlint-plugin.cabal [hunk 3]
```diff
@@ -2,11 +2,11 @@ cabal-version: 2.2
 name:          hls-hlint-plugin
 version:       0.1.0.0
 synopsis:      Hlint integration plugin with Haskell Language Server
-description:   Please see README.md
+description:   Please see Haskell Language Server Readme (https://github.com/haskell/haskell-language-server#readme)
 license:       Apache-2.0
 license-file:  LICENSE
 author:        Many,TBD when we release
```
Comment (jneira) [inline plugins/hls-hlint-plugin/hls-hlint-plugin.cabal:8]: I guess we should to change this one for sure 😄 
If we got a collective description, we could use it here

File: plugins/hls-hlint-plugin/hls-hlint-plugin.cabal (hunk 1 above)
Comment (alanz) [reply plugins/hls-hlint-plugin/hls-hlint-plugin.cabal:9]: Maybe just put the github repo there?

Comment (jneira) [reply plugins/hls-hlint-plugin/hls-hlint-plugin.cabal:9]: mmm, that could work

**Pull request merged**